### DISCUSSION TITLE
Some feature/plugin/config names are not obvious to users

### DISCUSSION THREAD

**Discussion by romainmenke** [csstools/postcss-plugins#86] [Ideas]
see : https://github.com/csstools/postcss-preset-env/issues/124

Being a 100% clear for all users when choosing feature/plugin/config names is not realistic.
Changing a vague name to something clearer is almost always a breaking change.

Can we do more with documentation?

A readme with a "dictionary" might already help a lot?

**Comment by Antonio-Laguna** [comment]
I've been thinking a lot about this and wanted to alleviate this too in a future website. The current one is a bit lax in this sense and I feel we can get something like Algolia on page with lots of synonyms to ease all of this.

**Reply by romainmenke** [reply]
Feedback in `postcss-preset-env` when passing an unknown feature name would also be great I think.

When a user sets `postcss-font-variant` the feedback might be : 
> Features are not referenced by their plugin name.
> Did you mean `font-variant-property`

**Reply by Antonio-Laguna** [reply]
That'd be awesome!
